modules/localization/msf/local_pyramid_map/pyramid_map/pyramid_map_config_test.cc

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#include "modules/localization/msf/local_map/pyramid_map/pyramid_map_config.h"
+#include <gtest/gtest.h>
+#include "modules/localization/msf/local_map/base_map/base_map_config.h"
+
+int main(int argc, char** argv) {
+  testing::InitGoogleTest(&argc, argv);
+  return RUN_ALL_TESTS();
+}
+
+namespace apollo {
+namespace localization {
+namespace msf {
+
+class PyramidMapConfigTestSuite : public ::testing::Test {
+ protected:
+  PyramidMapConfigTestSuite() {}
+  virtual ~PyramidMapConfigTestSuite() {}
+  virtual void SetUp() {}
+  virtual void TearDown() {}
+
+ protected:
```

Comment:
Remove unused marker.
And this Class does nothing. You can simply remove it and change the `TEST_F` to `TEST`.

Reply:
fixed